plugins/woocommerce/includes/admin/list-tables/class-wc-admin-list-table-orders.php

```diff
@@ -662,5 +662,23 @@ public function search_custom_fields( $wp ) {
 			// Search by found posts.
 			$wp->query_vars['post__in'] = array_merge( $post_ids, array( 0 ) );
 		}
+
+		if ( isset( $_GET['m'] ) ) {
+			$date_option = get_option( 'woocommerce_date_type', 'date_paid' );
+			$date_query  = wc_clean( wp_unslash( $_GET['m'] ) );
+
+			unset( $wp->query_vars['m'] );
```

Comment:
We should move this line inside of the below if statement, as this breaks the original behaviour showing all the orders when we have the date type set to `date_created`.